### PR TITLE
#2493   invariant clean up

### PR DESCRIPTION
**Pull request by sheridancook**
Commits address issues listed under this ticket: 
https://simplifier.net/cabaseline/patientprofile/~issues/2493

The only outstanding item not covered in this PR is the removal of smoking status which will resolve the outstanding invariant error but will have its own PR

### CODE REVIEW

**Comment by sheridancook** [comment]
@ElliotSilver  - can you review so we can merge PR - see note about smoking status inv being fixed through another PR (since it's being removed)

File: input/resources/structuredefinition-profile-observation-results.json [hunk 1]
```diff
@@ -372,7 +348,7 @@
       {
         "id": "Observation.component.value[x]:valueQuantity",
         "path": "Observation.component.value[x]",
-        "sliceName": "valueQuantity",
+        "sliceName": "valueQuantity",
```
Comment (ElliotSilver) [inline input/resources/structuredefinition-profile-observation-results.json:351]: Consider removing end of line whitespace.

**Review by ElliotSilver — APPROVED** [review]
Looks good. One minor tweak suggested before merging.

File: input/resources/structuredefinition-profile-observation-results.json (hunk 1 above)
Comment (sheridancook) [reply input/resources/structuredefinition-profile-observation-results.json:351]: Removed whitespace

**Pull request merged**